x-pack/plugins/cases/public/components/all_cases/columns.tsx

```diff
@@ -251,6 +255,23 @@ export const useCasesColumns = ({
           ? renderStringField(`${totalAlerts}`, `case-table-column-alertsCount`)
           : getEmptyTagValue(),
     },
+    ...(showSolutionColumn
+      ? [
+          {
+            align: RIGHT_ALIGNMENT,
+            field: 'owner',
+            name: i18n.SOLUTION,
+            render: (caseOwner: CasesOwners) => {
+              const ownerInfo = OWNER_INFO[caseOwner];
+              return caseOwner ? (
```

Comment:
I think we should check for the `ownerInfo` here.

Reply:
oh woops! Thay was the intent, but I changed the names of the variables around and missed that one. Thanks for catching it!